src/vs/workbench/services/backup/node/backupFileService.ts

```diff
@@ -172,6 +187,29 @@ export class BackupFileService implements IBackupFileService {
 		});
 	}
 
+	public getWorkspaceTextFileBackups(): TPromise<string[]> {
```

Comment:
@Tyriar at first I thought this should move into the cache but then we would stop each and every file open waiting for this to be done. Since backups can be restored later, I think it is OK to leave this outside of the cache even though the result could probably be cached.